commands/util/pvp-rank.js

```diff
@@ -0,0 +1,161 @@
+const log = require('loglevel').getLogger('Counters'),
+  Commando = require('discord.js-commando'),
+  {MessageEmbed} = require('discord.js'),
+  {CommandGroup} = require('../../app/constants'),
+  fetch = require('node-fetch'),
+  cheerio = require('cheerio'),
+  Utility = require('../../app/utility');
+
+class PvPRankingData {
+  constructor(pokemon, league, atk, def, sta, filter){
+    this.inputPokemon = pokemon;
+    this.inputLeague = league;
+    this.inputATKiv = atk;
+    this.inputDEFiv = def;
+    this.inputSTAiv = sta;
+    if(typeof(filter) == 'undefined'){
+      this.inputFilter = "0";
+    }
+    else{
+      this.inputFilter = filter;
+    }
+  }
+
+  async scrape(){
+    String.prototype.titleCase = function () {
+      return this.replace(/\w\S*/g, function(str) {return str.charAt(0).toUpperCase() + str.substr(1).toLowerCase();});
+    };
+    this.cpLeague;
+    let ivFilter;
+    if(this.inputLeague == 'ultra'){
+      this.cpLeague = "2500";
+    }
+    else {
+      this.cpLeague = "1500";
+    }
+    if(this.inputFilter != "10"){
+     ivFilter = "0"
+    }
+    this.url =
+    `https://gostadium.club/pvp/iv?pokemon=${this.inputPokemon}` +
+    `&max_cp=${this.cpLeague}` +
+    `&min_iv=${ivFilter}` +
+    `&att_iv=${this.inputATKiv}` +
+    `&def_iv=${this.inputDEFiv}` +
+    `&sta_iv=${this.inputSTAiv}`;
+
+    let $ = await fetch(this.url)
+      .then(res => res.text())
+      .then(body => cheerio.load(body))
+      .catch(err => log.error(err));
+    let ivScrapeData = [];
+    let siteData;
+    $("tr[class^=\"table-\"]").children().each(function(){

```

Comment:
```javascript
let ivScrapeData = $('tr[class^="table-"]').children().map((i, el) => $(el).text()).get();
```

Eliminates the need to cycle through, create a local variable and push the data to a separate array. (Its been a while since I've used Cheerio but I believe by looking at the docs this would work with `.get()`.)

Reply:
I'll test this out! Thanks!